howto_conda_package.md

```diff
@@ -0,0 +1,361 @@
+```
+git checkout <packagename>_recipe
+git add <package_name>
+git commit -m 'Meaningful message'
+git push origin <packagename>_recipe
+```
+You may always check the differences and your active branch using ``git status`` or ``git branch``.
+
+9. You are almost there! Now its time to request merging.
+  - Go to your branch on the github webpage. There should be something written like "branch is X commits ahead of bioconda-recipes:master". On the right-hand side, click on ``Pull request``.
+  - When the request is successful, bioconda will automatically build your package from your recipe and makes it available over the bioconda channel.
+  - To reach a successful pull request, you need to pass the bioconda test bot and at least one verified manual review by another bioconda user. 
+  **Bioconda test bot**:
+  It will perform three tests: linting, test-linux and test-macos. Most likely, those tests will fail the first time to try. If you are lucky, the manual review by another user may help you passing those tests.
+  However, if you want to test your package locally, consider the [local testings](https://bioconda.github.io/contributor/building-locally.html#using-the-bootstrap-method) of bioconda.
+  Those steps might become rather tricky and to the point as I write this protocol, HyPro did not pass the test-linux step. I suggest to get help from experienced bioconda reviewers, maybe the person that checked your recipe if the testing does not give sufficient info about the problem.
+
+## Packaging in Practice: An Example
+
+In this section I want to make the recipe I created transparent and comment on each file I used. This shall give you more info on how things are ment in the online documentary of conda. Note that I only used a few features and settings possible for setting up a package, especially in the meta.yaml.
+
+### **Building the meta.yaml file**
+
+This is the heart piece of your recipe. Please find the general structure and a first example [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html). Further this website provides **all options** on the meta.yaml, sorted per section. Most sections have many more options available than the ones mentioned. Please visit the website for these information.
+Please always mind the exact format - code indentations, whitespaces etc. might all have a meaning so keep the format straight!
+
+#### **The Header**
+```
+{% set name = "hypro" %}
+{% set version = "0.1" %}
+```
+
+Those are the first two lines - here, you specify the package name and package version. Mind the format!
+
+#### **The Package Section**
+```
+package:
+  name: hypro
+  version: {{ version }}
+```
+Specify the package information here. Set the version number in double-quotes, so it cannot be interpretet as float, whatever number you choose.
+
+#### **The Source Section**
+```
+  url: https://github.com/hoelzer-lab/hypro/archive/v{{ version }}.tar.gz

```

Comment:
I changed

```
  url: https://github.com/hoelzer-lab/hypro/archive/v0.1.tar.gz
```

to 

```
  url: https://github.com/hoelzer-lab/hypro/archive/v{{ version }}.tar.gz
```

but to be honest I am not sure if it works like this. Please review and revert in case it's not good